interfaces/apparmor/spec.go

```diff
@@ -524,7 +524,8 @@ func GenWritableMimicProfile(emit func(f string, args ...interface{}), path stri
 	emit("  # .. permissions for traversing the prefix that is assumed to exist\n")
 	for iter.Next() {
 		if iter.Depth() < assumedPrefixDepth {
-			emit("  \"%s\" r,\n", iter.CurrentPath())
+			cp := iter.CurrentPathPlusSlash()
```

Comment:
Initially I was worried that this is not a correct change but on some reconsideration I think this _is_ okay. While `CurrentPath` could return a string without a trailing slash, in practice it only did that for the leaf element.